cypress/fixtures/treesFilter_Mock.json

```diff
@@ -0,0 +1,427 @@
+[
+  {
+    "id": 666666,
+    "timeCreated": "+052341-09-10T13:56:40.000Z",
+    "planterId": 5290,
+    "active": true,
+    "planterIdentifier": "arunbakt@live.com",
```

Comment:
I'd suggest using made up email addresses or phone numbers here.

Reply:
The email addresses have been modified.